### PR TITLE
Release 3.1.3

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.1.3 (2019-11-19)
+
+- Fixed the summary action bar component to properly style the correct modal when two modals are on a page but only one has a summary action bar. [#25](https://github.com/blackbaud/skyux-action-bars/pull/25)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: Instead of `properly style`, should this say something more specific like `applies the proper padding to the footer`?

Comment (blackbaud-johnly) [reply CHANGELOG.md]: I like that. What do you guys think of:
`Fixed the summary action bar component to apply the proper padding in modal footers when two modals are open and only one has a summary action bar.`?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Ok. I made a change. How does it look now?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: @blackbaud-johnly technically we are removing padding and we only want to do it for one modal footer when this happens. Any way you could glance at the change I made and maybe tweak that to how you like it now?

**Pull request merged**